packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.test.tsx

```diff
@@ -31,4 +32,77 @@ describe("Chip Compound Components", () => {
     const elem = container.querySelector("." + styles.empty);
     expect(elem).toBeInTheDocument();
   });
+
+  describe("when onClick prop is passed", () => {
+    it("should call onClick when suffix is clicked", () => {
+      const onClick = jest.fn();
+      const { getByTestId } = render(
+        <Chip.Suffix onClick={onClick} testID="ATL-Chip-Suffix">
+          <Icon name="cross" />
+        </Chip.Suffix>,
+      );
+
+      getByTestId("ATL-Chip-Suffix")?.click();
+      expect(onClick).toHaveBeenCalledTimes(1);
+    });
```

Comment:
I think it would be nice to use `userEvents` here (and more often in general), since this is considered a standard way to interact with elements by RTL (and is similar to what you do below when testing key press):

```suggestion
    it("should call onClick when suffix is clicked", async () => {
      const onClick = jest.fn();
      const { getByTestId } = render(
        <Chip.Suffix onClick={onClick} testID="ATL-Chip-Suffix">
          <Icon name="cross" />
        </Chip.Suffix>,
      );

      await userEvent.click(getByTestId("ATL-Chip-Suffix"));
      expect(onClick).toHaveBeenCalledTimes(1);
    });
```

Reply:
Thanks for pointing that out! I just talked with Jacob recently about using `userEvent` more frequently and didn't make the connection that I could use it here also.